dev-util/cutter/cutter-1.11.1.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8,9} )
+
+inherit qmake-utils xdg-utils python-single-r1
+
+DESCRIPTION="A Qt and C++ GUI for radare2 reverse engineering framework"
+HOMEPAGE="https://www.radare.org https://github.com/radareorg/cutter/"
```

Comment:
I would suggest to change home page to official https://cutter.re/

Reply:
Done, thanks!